### DISCUSSION TITLE
Add warning or restrict multiplayer option if playing on non latest version.

### DISCUSSION THREAD

**Discussion by Cootz** [ppy/osu#22442] [Ideas] [answered]
Recently i've played osu lazer with my friend. We hopped into multiplayer and selected maps to play. But as we try to start a map, i was the only one playing. Osu shows that she quit the map when in reallity she did not. Turns out that she was playing on non latest version and didn't even notice that. Osu doesn't provide any info about this issue to user. With that said I suggest osu needs to notify player about the version difference (or non latest version) when he/she tries to play multi.

**Comment by WindowsMeosu** [comment]
I'll start this off that, if I remember correctly,  I am 100% pretty sure you are notified when there is a new release of lazer and also I feel restricting multiplayer on old versions of lazer is no easy job. I'll also mention that some people could have a specific reason to use or switch to a outdated client and disabling multiplayer on them seems to go above the boundaries. It's possible that this was caused from something else entirely unrelated to multiplayer or outdated clients but I don't think going as far to disable multiplayer on said clients is a really good idea.

**Reply by Cootz** [reply]
Mb I wrote it the wrong way. I mean that avarrage user have no clue that playing multi requiers same version for every player in the room. Just warning about version difference for the room with suggestion to update to latest version would be enought.

**Reply by Cootz** [reply]
Surely its non an issue for devs, but it is for an avarage user

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu-server-spectator/issues/52
